src/webots/gui/WbMainWindow.hpp

```diff
@@ -150,7 +153,13 @@ private slots:
   void uploadFinished();
 
 private:
-  void showHtmlRobotWindow(WbRobot *);
+  void showHtmlRobotWindow(WbRobot *robot);
+  void closeClientRobotWindow(WbRobot *robot);
+  void onSocketOpened();
+  QList<WbRobotWindow *> mRobotWindows;
+  QList<WbRobot *> mPendingRobots;
```

Comment:
This may be a better name?
```suggestion
  QList<WbRobot *> mRobotsWaitingForWindowToOpen;
```